sopel/tools/jobs.py

```diff
@@ -33,7 +33,14 @@ def __exit__(self, _type, _value, _traceback):
 
 
 class PriorityQueue(Queue.PriorityQueue):
-    """A priority queue with a peek method."""
+    """A priority queue with a peek method.
+
+    .. deprecated:: 7.0
+
+        This will be removed in Sopel 8. Use Python's built-in priority queue
+        instead.
+
```

Comment:
Blank line left here when the others like it in the file were removed? :confused:
```suggestion
```

(Or does leaving out this particular blank line break something in Sphinx?)